guides/slashing_and_withdrawals.md

```diff
@@ -0,0 +1,221 @@
+# Slashing & Withdrawals
```

Comment:
@MatheusFranco99 @riccardo-ssvlabs please have a look at this guide to check if I missed something or it can be improved. thanks!